chef/cookbooks/monasca/recipes/role_monasca_log_api.rb

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
yeah, will revert that. Guess it might be simpler to run everything from one point.
If necessary, we can split it up.